prow/jobs/kyma/kyma-integration.yaml

```diff
@@ -74,6 +74,34 @@ postsubmits:
           - image: eu.gcr.io/kyma-project/prow/bootstrap:0.0.1
             command:
               - "/home/prow/go/src/github.com/kyma-project/test-infra/prow/scripts/provision-vm-and-start-kyma.sh"
+    - name: kyma-gke-integration-master
```

Comment:
in the Migration guide, we have the same name for presubmit and postsubmit job. Is there any reason to keep sufix master?

Reply:
The existing presubmit jobs are `kyma-integration` and `kyma-gke-integration`, and the existing postsubmit job was `kyma-integration-master`. I created `kyma-gke-integration-master` as it seemed consistent with the existing naming

Reply:
I understand that, but maybe in your PR we can make it consistent with other jobs?